packages/core/helper-plugin/package.json

```diff
@@ -46,8 +46,10 @@
     "axios": "1.2.2",
     "date-fns": "2.29.3",
     "formik": "^2.2.6",
+    "fs-extra": "10.0.0",
```

Comment:
this should be in the devDeps as its used in development to generate the build index (although we no longer need this so it can be removed – should be a separate PR though).

Reply:
This PR https://github.com/strapi/strapi/pull/15797 lets us remove this completely! Fantastic!

Related log: https://gist.github.com/smorimoto/07f7df255735bb250d3116565261f24d#file-unsafe-packages-json-L933

Reply:
Done.